### ISSUE TITLE
Increase Rows and Columns in the Start Menu folders

### ISSUE THREAD

**Issue by Khalylexe** [ramensoftware/windows-11-start-menu-styling-guide#74]
Can we increase the amount of rows and columns in a folder?
also is it possible to show more than just four icons in the folder icon

**Comment by bbmaster123** [comment]
yes you can increase rows and columns in the expanded folder view, but keep in mind it can mess with scrolling behavior and may cause some of the icons to appear cut off. With some trial and error it might be possible to find and set some values that work without causing any side effects
```
Target: StartMenu.ExpandedFolderList > Grid#Root > Border
Style: Height=500
```

use uwpspy to find parent, child and sibling elements in the visual tree to target to adjust the rest of the expanded folder list (header position, borders, etc)

It is not possible as far as I know to change the number of icons in the folder grid, we would need to be able to access the external ItemsRepeater.Layout named StartMenu.RowMajorUniformToFirstGridLayout which has properties MaxColumns=2 and MaxRows=2. this is referenced by the element ItemsRepeater#LogosRepeater under Layout=
unfortunately I don't see a way to access and modify those within the styler mod

hope this helps!
